internal/node/startup/startup.go

```diff
@@ -19,10 +20,23 @@ import (
 )
 
 // Validates the Node Database Schema Version
-func ValidateSchema(postgresEndpoint string) error {
-	schemaManager, err := repository.NewSchemaManager(postgresEndpoint)
-	if err != nil {
-		return err
+func ValidateSchema(config config.NodeConfig) error {
+	var (
+		schemaManager *repository.SchemaManager
+		err           error
+	)
+
+	if strings.Contains(config.BlockchainHttpEndpoint.Value, "devnet") {
```

Comment:
I'm not sure that checking for [`devnet`](https://github.com/cartesi/rollups-node/blob/584edb0145ae17efbea577f2114684c54dddf773/build/compose-devnet.yaml#L18) is robust enough. Perhaps we should add an extra config parameter to explicitly ask to disable SSL.

Reply:
I agree